OPAL/da/src/main/scala/org/opalj/da/PermittedSubclasses_attribute.scala

```diff
@@ -0,0 +1,29 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.da
+
+import scala.xml.{Node, NodeBuffer}
+
+/**
+ * Represents the ''PermittedSubclasses'' attribute (Java 17).
+ *
+ * @author Julius Naeumann
+ */
+case class PermittedSubclasses_attribute(
+        attribute_name_index: Constant_Pool_Index,
+        permittedSubclasses:  ClassesArray
```

Comment:
I think everything else here in DA uses underscores instead of camel case, so maybe we should stick to that

Reply:
I'm referring to the name of the field: I think it should be permitted_subclasses to match the style of other attribute classes.

Reply:
Ok, I will change it to camel case. The other attribute classes seem to be mixing camel case and underscores.